TwilioStore.Tests/Services/NotificationServiceTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using TwilioStore.Interfaces.Exceptions;
+using TwilioStore.Interfaces.Services;
+using TwilioStore.Services;
+
+namespace TwilioStore.Tests.Services
+{
+    [TestClass]
+    public class NotificationServiceTest
+    {
+        // Get your test credentials from https://www.twilio.com/console/account/settings
+        private const string TestAccountSid = "ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
```

Comment:
I'd like to see some instruction that warn me about that I have to update the test credentials to make it work. The first time I ran the test it fails, fortunately the error message was descriptive enough to tell me I have to update the current values with my credentials.